doc/reference/algorithms/community.rst

```diff
@@ -58,6 +58,14 @@ Label propagation
    label_propagation_communities
    fast_label_propagation_communities
 
+Local Community Detection
+-------------------------
+.. automodule:: networkx.algorithms.community.local
+.. autosummary::
+   :toctree: generated/
+
+   clauset
```

Comment:
Minor note: Once we've settled on the function name, this will have to match it!